proto/ibc/core/channel/v1/tx.proto

```diff
@@ -81,6 +81,7 @@ message MsgChannelOpenTry {
   string port_id = 1 [(gogoproto.moretags) = "yaml:\"port_id\""];
   // in the case of crossing hello's, when both chains call OpenInit, we need
   // the channel identifier of the previous channel in state INIT
+  // this field has been deprecated and will be ignored by core IBC becase now we don't need protect against crossing hellos anymore .
```

Comment:
```suggestion
  // this field has been deprecated and will be ignored by core IBC becase now we don't need to protect against crossing hellos anymore .
```

I think this is actually just true for our implementation because we autogenerate our channel IDs but this is not necessarily true for other fields. Is there a better way we should explain this here @colin-axner ?